tests/conftest.py

```diff
@@ -11,3 +14,20 @@ def disable_insecure_requests_warning():
 @pytest.fixture(scope="session", autouse=True)
 def load_local_cluster_kubeconfig():
     config.load_kube_config('./kubeconfig')
+
+
+@pytest.fixture
+def temporary_namespace():
```

Comment:
You can call it namespace or k8s_namespace. The fact that it is temporary is embedded in the fixture itself.

And do you really need it at a test level scope? Won't a session level be good? If session level, you can even use `autouse` to avoid mentioning it as a dependency in the test modules.

Reply:
I don't want to handle test cleanup myself. I'd keep it this way and change it once we have more tests.